src/components/error-boundary.tsx

```diff
@@ -9,17 +9,21 @@ type State = {
 };
 
 export class ErrorBoundary extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { error: null };
+  }
 
   static getDerivedStateFromError(error: Error) {
-    return {error}
+    return { error };
   }
 
   componentDidCatch(error: Error, _: React.ErrorInfo) {
-    this.setState({error: error});
+    this.setState({ error: error });
   }
 
   render() {
-    const {error} = this.state
+    const { error } = this.state || { error: null };
```

Comment:
Do we still need that evenso state is properly initialized?

Reply:
probably not! removed it.